src/PrestaShopBundle/Form/Admin/Type/TextPreviewType.php

```diff
@@ -38,7 +38,7 @@
  * It is based on a hidden input so it could be changed programmatically, or be used just to display
  * a data in a form.
  */
-class TextPreviewType extends HiddenType
+class TextPreviewType extends TextType
```

Comment:
What was wrong with this?

Reply:
this commit was done by @jolelievre . He helped to improve it so @jolelievre maybe you can comment on why this was changed?

Reply:
Basically, hidden inputs have no errors shown, and yet we define some constraints on this field in `DeltaQuantityType` (and potentially other future form types). By using a TextType the `form_errors` works, but the input itself is still defined as a hidden input in the layout (because we force `$view->vars['type'] = 'hidden';` in buildView) so it's still hidden and shouldn't be edited manually